src/templates/doe_workflow_central_ducted_split_heat_pump.mdx

```diff
@@ -107,7 +109,7 @@
         <NumberInput label="Dual Fuel Switch Over Temperature (&deg;F)" min={0} path="dual_fuel_switch_over_temperature" /> 
 
         <PhotoInput id="mfg_extended_cooling_performance_table_photo" label="MFG's cooling performance table – Photo" uploadable>
-            Upload a photo of the MFG's extended cooling performance table showing sensible, latent, or SHR.
+            Upload a photo of the MFG's extended cooling performance table showing sensible, latent, or SHR. 
```

Comment:
See https://writingexplained.org/english-abbreviations/manufacturer for the correct way to abbreviate manufacturer.  In the label, it should be "Mfr's". We should write out "manufacturer's" in the description.

Can you search through the template for other instances of this?

Reply:
updated accordingly in label and description